src/MySQLReplication/Event/RowEvent/RowEvent.php

```diff
@@ -662,7 +662,7 @@ protected function getDecimal(ColumnDTO $columnDTO): string
             $res .= sprintf('%0' . $compFractional . 'd', $value);
         }
 
-        return bcmul($res, '1', $columnDTO->getPrecision());
+        return bcmul($res, '1', $columnDTO->getDecimals());
```

Comment:
yep decimal = scale 
in php 7.3 => they fix bc that now correctly add missing 0 for scale